bootstrap/shell/v001__ingest_defaults.sh

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env bash
```

Comment:
Can you give execute permissions for this and any other shell scripts in github..
when I run I see the error like below

```

Script failed
-------------
SQL State  : null
Error Code : 0
Message    : Failed to run script "/Users/aiyer/github/streamline/./bootstrap/shell/v001__ingest_defaults.sh", Cannot run program "/Users/aiyer/github/streamline/./bootstrap/shell/v001__ingest_defaults.sh": error=13, Permission denied
```

Reply:
Done